components/collective-page/CollectiveNotificationBar.js

```diff
@@ -85,11 +85,21 @@ const getNotification = (intl, status, collective, host) => {
 /**
  * Adds a notification bar for the collective.
  */
-const CollectiveNotificationBar = ({ intl, status, collective, host }) => {
+const CollectiveNotificationBar = ({ intl, status, collective, host, LoggedInUser }) => {
+  // Quickfix for events (as they are not inheriting isApproved status)
+  if (collective.type === CollectiveType.EVENT) return null;
```

Comment:
Not using curly braces generates an ESLint warning (see https://eslint.org/docs/rules/curly). If you're using VSCode, I recommend installing https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint to quickly spot them (other editors usually provide extensions like that too).

Anyway I don't understand why hiding **all** notifications for events, could you provide more details?

Reply:
@JoyLubega can you answer this one?